kairon/shared/channels/whatsapp/bsp/dialog360.py

```
@@ -366,6 +390,54 @@ def _do():
+            ).only("filename", "media_id", "upload_status", "sender_id", "timestamp")
+            return [
+                {
+                    "filename": doc.filename,
+                    "media_id": doc.media_id,
+                    "upload_status": doc.upload_status,
+                    "sender_id": doc.sender_id,
+                    "timestamp": doc.timestamp,
+                }
+                for doc in media_data
+            ]
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)